pkg/tsdb/phlare/instance.go

```diff
@@ -40,8 +41,17 @@ func NewPhlareDatasource(httpClientProvider httpclient.Provider, settings backen
 		return nil, err
 	}
 
+	sourceHTTPClient, err := httpClientProvider.New()
+	if err != nil {
+		return nil, err
+	}
+
 	return &PhlareDatasource{
 		client: querierv1connect.NewQuerierServiceClient(httpClient, settings.URL),
+		sourceCodeClient: &sourceCodeClient{
+			client:  sourceHTTPClient,
+			baseURL: "https://go-source.swine.dev/source/go",
```

Comment:
We probably should just do this locally with `http://localhost:8090` and run a container in the pod with https://github.com/simonswine/go-source-server

Reply:
Yep a side car on grafana will work I guess